tests/SpecHook.hs

```diff
@@ -0,0 +1,11 @@
+module SpecHook
+  ( hook
+  ) where
+
+import Prelude
+
+import Test.Hspec
+import Test.Hspec.JUnit.Formatter.Env as Formatter
+
+hook :: Spec -> Spec
+hook = Formatter.whenEnabled Formatter.add . parallel
```

Comment:
:ok_hand:

Reply:
Unfortunately it requires at minimum LTS 20.

Reply:
Nevermind, just required https://github.com/freckle/hspec-junit-formatter/pull/30